### PR TITLE
Automation Debugging

### PR DESCRIPTION
**Pull request by knudsvik**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [x] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @rolfikr1, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#18400).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by frenck** [comment]
There are unrelated commits in this PR. Please perform a git rebase onto the latest upstream current branch, to clean those up:

Thanks!

**Review by frenck — CHANGES_REQUESTED** [review]
Setting PR state correctly

File: source/_docs/automation/troubleshooting.markdown [hunk 1]
```diff
@@ -3,18 +3,27 @@ title: "Troubleshooting Automations"
 description: "Tips on how to troubleshoot your automations."
 ---
 
-You can verify that your automation rules are being initialized correctly by watching both the realtime logs (`homeassistant.log` in the configuration directory) and also the [Logbook](/integrations/logbook/). The realtime logs will show the rules being initialized (once for each trigger), example:
+### Debugging your automation
```
Comment (frenck) [inline source/_docs/automation/troubleshooting.markdown]: This is the page title already, so we don't need another heading.

```suggestion
```

File: source/_docs/automation/troubleshooting.markdown [hunk 2]
```diff
@@ -3,18 +3,27 @@ title: "Troubleshooting Automations"
...
 
-```text
-INFO [homeassistant.components.automation] Initialized rule Rainy Day
-INFO [homeassistant.components.automation] Initialized rule Rainy Day
-INFO [homeassistant.components.automation] Initialized rule Rainy Day
-INFO [homeassistant.components.automation] Initialized rule Rain is over
-```
+Automation debugging was added in the 2021.4 release. When an automation is run, all steps are recorded and a trace is made. From the UI choose **Configuration** which is located in the sidebar, then click on **Automation** to go to the automation editor or click this button directly:  [![Open your Home Assistant instance and show your automations.](https://my.home-assistant.io/badges/automations.svg)](https://my.home-assistant.io/redirect/automations/)
```
Comment (frenck) [inline source/_docs/automation/troubleshooting.markdown]: We don't keep a changelog in the documentation, so mentioning the version isn't needed.

```suggestion
When an automation is run, all steps are recorded and a trace is made. From the UI choose **Configuration** which is located in the sidebar, then click on **Automation** to go to the automation editor or click this button directly: {% my automation badge %}
```

File: source/_docs/automation/troubleshooting.markdown [hunk 3]
```diff
@@ -3,18 +3,27 @@ title: "Troubleshooting Automations"
...
+
+
+Click on the clock icon next to an automation to enter the debugging screen. Alternatively click on **Show trace** directly from a Logbook automation entry.
```
Comment (frenck) [inline source/_docs/automation/troubleshooting.markdown]:
```suggestion
Click on the clock icon next to an automation to enter the debugging screen. Alternatively, click on **Show trace** directly from a Logbook automation entry.
```

File: source/_docs/automation/troubleshooting.markdown [hunk 4]
```diff
@@ -3,18 +3,27 @@ title: "Troubleshooting Automations"
...
+
+![Automation tracing example](/images/integrations/automation/automation-tracing.png)
 
-The Logbook integration will show a line entry when an automation is triggered. You can look at the previous entry to determine which trigger in the rule triggered the event.
+The above screenshot shows a previous run of an automation. The automation is displayed using an interactive graph, highlighting which path the automation took. Each node in the graph can be clicked to view the details on what happened with the automation during that specific step. It traces the complete run of an automation.
 
-![Logbook example](/images/integrations/automation/logbook.png)
+The debugging screen is split in four features, the first being the Step Details which provides all details for each step of the automation. The second feature is the Trace Timeline which the screenshot above shows and where the automation can be followed on a timeline. The next is Related logbook entries, as the name says a logbook for all the entries related to the specific trace. The last two features are Automation Config and optionally Blueprint Config for the automation YAML code.
```
Comment (frenck) [inline source/_docs/automation/troubleshooting.markdown]:
```suggestion
The debugging screen is split into four features, the first being the Step Details which provides all details for each step of the automation. The second feature is the Trace Timeline which the screenshot above shows and where the automation can be followed on a timeline. The next is Related logbook entries, as the name says a logbook for all the entries related to the specific trace. The last two features are Automation Config and optionally Blueprint Config for the automation YAML code.
```

**Review by frenck — CHANGES_REQUESTED** [review]
Looks good @knudsvik  👍 

All that is left is a couple of small comments and a merge conflict that needs to be resolved.

**Review by frenck — APPROVED** [review]
Thanks, @knudsvik 👍

**Pull request merged**